microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -72,131 +61,129 @@ The default channel-search order is:
 
-To make your application use a Microsoft Edge preview channel by using a registry override that sets the browser executable folder:
+* Use the built-in policy on Intune. Refer to [Configure Microsoft Edge policy settings in Microsoft Intune](https://learn.microsoft.com/en-us/mem/intune/configuration/administrative-templates-configure-edge) for more information.
 
-1. Open a PowerShell terminal or PowerShell-enabled command prompt.
+---
 
-1. Modify and then run the following command:
 
-   `REG ADD HKLM\Software\Policies\Microsoft\Edge\WebView2\BrowserExecutableFolder /v WebView2APISample.exe /t REG_SZ /d "C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0"`
+### Browser executable folder
+
+One approach is to use a browser executable folder.  In this approach, you specify a folder that contains runtime binaries. This folder can be any of the following locations:
+*  The installed location of the WebView2 Runtime.
+*  A preview channel of Microsoft Edge.
+*  A folder containing fixed version binaries that you have deployed to the machine yourself.
+
+If you set the browser executable folder to a specific preview channel of Microsoft Edge, you will need to update the location when that preview channel updates to a newer version. This is because the location includes the version number as part of its path. Therefore, we recommend using this approach for local testing only.
 
-   Replace `WebView2APISample.exe` with the file name of your own app executable or the Application User Model ID. Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine and can result in unexpected behavior.
+##### [API](#tab/api)
 
-   Replace `C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0` by the path to the desired Microsoft Edge preview channel.
+Pass the path to the browser executable folder during environment creation with the following APIs
 
-#### Resuming using the default, WebView2 Evergreen Runtime
+.NET (WPF):
+* [CreationProperties Property](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.wpf.webview2.creationproperties)
+* [CoreWebView2CreationProperties Class](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.wpf.corewebview2creationproperties)
 
-To undo the above setting, run the following command:
+.NET (WinForms):
+* [CreationProperties Property](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.winforms.webview2.creationproperties)
+* [CoreWebView2CreationProperties Class](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.winforms.corewebview2creationproperties)
 
-`REG DELETE HKLM\Software\Policies\Microsoft\Edge\WebView2\BrowserExecutableFolder /f`
+WinRT:
+<!--todo: Unsure if there's a WinRT api, add information here -->
```

Comment:
@mikehoffms @vickiez I'm not sure if there's an equivalent for WinRT for the API

Reply:
@peiche-jessica - is there a WinRT API?

Pass the path to the browser executable folder during environment creation, by using the following APIs.

.NET (WPF):
* [CreationProperties Property](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.wpf.webview2.creationproperties)
* [CoreWebView2CreationProperties Class](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.wpf.corewebview2creationproperties)

.NET (WinForms):
* [CreationProperties Property](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.winforms.webview2.creationproperties)
* [CoreWebView2CreationProperties Class](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.winforms.corewebview2creationproperties)

WinRT:
* TBD<!--todo: link to WinRT API Ref, if exists -->

Win32:
* [CreateCoreWebView2EnvironmentWithOptions](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/webview2-idl?#createcorewebview2environmentwithoptions)

Reply:
It's a parameter to [CoreWebView2Environment.CreateWithOptionsAsync](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment?view=webview2-winrt-1.0.2420.47#createwithoptionsasync)

Reply:
```
WinRT:
* [CoreWebView2Environment.CreateWithOptionsAsync Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createwithoptionsasync)
```